gradle/dependencies.gradle

```diff
@@ -55,12 +55,13 @@ def support = [
 ]
 
 def test = [
-    junit4            : "junit:junit:4.12",
-    junit5Jupiter     : ["org.junit.jupiter:junit-jupiter-api:5.0.2","org.apiguardian:apiguardian-api:1.0.0"],
-    inferAnnotations  : "com.facebook.infer.annotation:infer-annotation:0.11.0",
-    cfQual            : "org.checkerframework:checker-qual:2.3.0",
-    cfCompatQual      : "org.checkerframework:checker-compat-qual:2.3.0",
-    rxjava2           : "io.reactivex.rxjava2:rxjava:2.1.2",
+    junit4                  : "junit:junit:4.12",
```

Comment:
Any particular reason for changing indentation here?

Reply:
Nope. Typo. Fixed locally. Will update PR.

Reply:
So we do need to change indentation because `jetbrainsAnnotations` is a long-ass name. But made it so that they all align fine.